sqltype/null_utc_time.go

```diff
@@ -0,0 +1,33 @@
+package sqltype
```

Comment:
i thought we said `sqltypes` ?

Reply:
yes but to be consistent we others folders which are in singular, no ?

Reply:
it is like, `errors` or `strings` when it it is a suite of tool without a main purpose ( a specific struct ) you can pluralize